torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -2429,8 +2403,9 @@ def _create_rref(self):
         )
 
     @dist_init
-    @_skip_if_tensorpipe_agent
     def test_user_rrefs_confirmed(self):
+        print(os.getpid())
+        # time.sleep(20)
```

Comment:
Oups, these slipped in from my debugging sessions, I'll remove them (and the import above) but I first want to wait for the CI to finish.